pbsmetrics/go_metrics.go

```diff
@@ -218,6 +228,10 @@ func NewMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderName, d
 
 	newMetrics.TimeoutNotificationSuccess = metrics.GetOrRegisterMeter("timeout_notification.ok", registry)
 	newMetrics.TimeoutNotificationFailure = metrics.GetOrRegisterMeter("timeout_notification.failed", registry)
+
+	newMetrics.TCF1Meter = metrics.GetOrRegisterMeter("privacy.tcf.v1", registry)
```

Comment:
I really like the new 'dotted' name format. Other metrics use a combination of underscore and dots. Thoughts on consistency?

Reply:
timeout_notification should remain underscore because it is a compound name, rather than notification being a subcategory of timeout.